api/routes.go

```diff
@@ -0,0 +1,32 @@
+package api
+
+import (
+	"github.com/Unknwon/macaron"
+	"github.com/macaron-contrib/binding"
+	"github.com/raintank/metrictank/api/middleware"
+	"github.com/raintank/metrictank/api/models"
+)
+
+func (s *Server) RegisterRoutes() {
+	r := s.Macaron
+	r.Use(macaron.Renderer())
+	r.Use(middleware.OrgMiddleware())
+	r.Use(middleware.CorsHandler())
+
+	bind := binding.Bind
+
+	r.Get("/", s.appStatus)
+	r.Get("/node", s.getNodeStatus)
+	r.Post("/node", bind(models.NodeStatus{}), s.setNodeStatus)
+
+	/* not sure what even uses this.
+	r.Combo("/get").Get(s.getMetrics).Post(s.getMetrics)
+	*/
```

Comment:
The current implementation in graphite_metrictank uses the best way possible to get data. 
/find does not return the metricIds and we dont really want it to so that we can have metrictank handle any merges when there are multiple metricDefs for a single series name.